AntennaTracker/GCS_Tracker.cpp

```diff
@@ -4,7 +4,6 @@
 void GCS_Tracker::request_datastream_position(const uint8_t sysid, const uint8_t compid)
 {
     for (uint8_t i=0; i < num_gcs(); i++) {
-        if (gcs().chan(i).initialised) {
             // request position
```

Comment:
Whitespace. There is a lot of off whitespace now...

Reply:
This was quite intentional to make it easier to review.  It  can be fixed with an NFC follow-up.